src/sagemaker_server.cc

```diff
@@ -772,8 +765,11 @@ SagemakerAPIServer::SageMakerMMEUnloadModel(
     LOG_ERROR << "Unable to unregister model repository for path: "
               << repo_parent_path << std::endl;
     TRITONSERVER_ErrorDelete(unload_err);
+  } else {
+    evhtp_send_reply(req, EVHTP_RES_OK);
   }
 
+  std::lock_guard<std::mutex> lock(mutex_);
```

Comment:
Just a comment, but it can be nice to name a mutex based on what it is intended to protect. `models_list_mutex_` would make it clearer what it is needed for.

Reply:
Thanks, done